js/domUpdates.js

```diff
@@ -1,59 +1,99 @@
-const players = [];
-const clues = [];
-const rounds = [];
-let game;
+const jeopardy = {
+  players: [],
+  clues: [],
+  rounds: [],
+  game: {},
+  categoryNames: {
+    unitedStatesHistory: 'United States History',
+    lifeSciences: 'Life Sciences',
+    publicHealth: 'Public Health',
+    educationJargon: 'Education Jargon',
+    nameThatBoardGame: 'Name That Board Game',
+    americanLiterature: 'American Literature',
+    biographies: 'Biographies',
+    americanCities: 'American Cities',
+    food: 'Food',
+    cableTV: 'Cable TV'
+  }
+};
 
-$('.input--submit').on('click', createPlayers);
-$('.main').on('click', presentClue);
+$('.input--submit').on('click', instantiatePlayers);
 
 function presentClue(event) {
-  console.log(event.target)
-  let colRow = event.target.dataset.colrow;
-  console.log(colRow);
-  let selectedClue = rounds[0].clues[parseInt(colRow)];
-  console.log(selectedClue)
-  alert(selectedClue.question);
+  if (event.target.innerHTML !== '') {
+    const id = event.target.dataset.id;
+    const selectedClue = jeopardy.rounds[0].clues[parseInt(id)];
+    if (selectedClue.dailyDouble === true) {
+      alert('DAILY DOUBLE!\n' + selectedClue.question);
+    } else {
+      alert(selectedClue.question);
+    }
+    event.target.innerHTML = '';
+    jeopardy.game.cluesRemaining--;
+  }
 }
 
-function createPlayers(event) {
+function instantiatePlayers(event) {
   event.preventDefault();
-  players.push(new Player($('#name-0').val()));
-  players.push(new Player($('#name-1').val()));
-  players.push(new Player($('#name-2').val()));
+  jeopardy.players.push(new Player($('#name-0').val()));
+  jeopardy.players.push(new Player($('#name-1').val()));
+  jeopardy.players.push(new Player($('#name-2').val()));
   updatePlayersOnDOM();
-  startGame();
+  this.disabled = true;
+  instantiateGame();
 }
 
 function updatePlayersOnDOM() {
-  players.forEach((player, index) => {
+  jeopardy.players.forEach((player, index) => {
     $(`.h4__player-${index}-name`).text(player.name); 
   });
 }
 
-function startGame() {
-  game = new Game(1, players[0])
-  $('.span--round').text(game.round);
-  getClues();
-  makeRounds();
+function instantiateGame() {
+  jeopardy.game = new Game(1, jeopardy.players[0])
+  $('.span--round').text(jeopardy.game.round);
+  instantiateClues();
+  instantiateRounds();
+  $('.main').on('click', presentClue);
 }
 
-function getClues() {
+function instantiateClues() {
   data.clues.forEach(clue => {
     const { question, answer, pointValue, categoryId } = clue;
-    clues.push(new Clue(question, answer, pointValue, categoryId));
+    jeopardy.clues.push(new Clue(question, answer, pointValue, categoryId));
```

Comment:
Clues are something that should probably be stored as an instance property of your game class, rather than just floating around in a global object literal that you created -- they are clues specifically tied to a particular instance of a game. What if (for some reason) you had 2 separate instances of a Jeopardy game on the page at once? How would each one know which clues belonged to which?